src/main/java/school/lemon/changerequest/java/introduction/pr1/ConsoleCalculator.java

```diff
@@ -1,9 +1,64 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
 
+import java.util.Scanner;
+
 public class ConsoleCalculator {
 
     public static void main(String[] args) {
+        listOfChoises();
+        Scanner sc = new Scanner(System.in);
+        int num1, num2;
+        while (true) {
+            System.out.println("Make your choice.");
+            String operation = sc.next().toLowerCase();
+            if (!operation.equals("exit, help, add, sub, div, mul")) {
+                System.out.println("Your type not ringt, you was trying - " + "/" + operation + "/" + " but bellow that you can do");
+                listOfChoises();
+                continue;
+            } else if (operation.equals("exit")) {
+                System.out.println("Bye-bye");
+                break;
+
+            } else if (operation.equals("help")) {
+                listOfChoises();
+                continue;
+            }
+            System.out.println("Enter first number:");
+            num1 = sc.nextInt();
```

Comment:
Please add `sc.nextLine();` after each `sc.nextInt()`. This method doesn't read line endings, so nextLine will return just `\n`.